module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -479,7 +479,7 @@ namespace module::behaviour::skills {
         // Rotate target angles to World space
         Eigen::Vector3d lookVector = headToIMUSpace * lookVectorFromHead;
         // Compute inverse kinematics for head direction angles
-        std::vector<std::pair<ServoID, double>> goalAngles = calculateCameraLookJoints(lookVector);
+        std::vector<std::pair<ServoID, float>> goalAngles = calculateHeadJoints(lookVector);
```

Comment:
Why was this changed to a float?

Reply:
Because `calculateHeadJoints` returns a float